lib/promise.js

```diff
@@ -2,5 +2,11 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+'use strict'
+
 // for easy promise lib switching
 module.exports = require('bluebird')
+
+process.on('unhandledRejection', (reason, promise) => {
```

Comment:
It feels like a little bit of a surprising place to put it is all I'm thinking. I get that it's quite promisey, but it's also quite processy/servery. I'm not that bothered about it tbh, just wanted to mention it.